docs/getting-started.md

```diff
@@ -127,10 +127,10 @@ sudo systemctl start docker
 # This will produce the firecracker and jailer binaries under
 # `./firecracker/build/cargo_target/${toolchain}/debug`.
 #
-sudo ./firecracker/tools/devtool build
+sudo ./firecracker_src/tools/devtool build
 
 # Rename the binary to "firecracker"
-mv ./firecracker/build/cargo_target/${ARCH}-unknown-linux-musl/debug/firecracker firecracker
+sudo cp ./firecracker_src/build/cargo_target/${ARCH}-unknown-linux-musl/debug/firecracker firecracker
```

Comment:
to be honest, I don't quite see why we need this step. Can we run the binary from where it was produced?

Reply:
there is an inconsistency between the steps. The below step downloads the release binary in the current folder
> mv release-${latest}-$(uname -m)/firecracker-${latest}-${ARCH} firecracker

so the below step to run firecracker from build doesn't work when we download the release binary.

># Run firecracker
>./firecracker/build/cargo_target/${ARCH}-unknown-linux-musl/debug/firecracker \
>    --api-sock "${API_SOCKET}"

Reply:
I see, it's to unify the call to the binary that is either built or downloaded...